packages/admin-panel/src/pages/resources/EntitiesPage.js

```diff
@@ -64,6 +64,14 @@ const IMPORT_CONFIG = {
         type: 'boolean',
       },
     },
+    {
+      label: 'Automatically fetch GeoJSON (defaults to "Yes")',
```

Comment:
Would be good if the default showed up as the checked box, rather than having to state it here in the label.